checkov/terraform/plan_parser.py

```diff
@@ -293,10 +315,14 @@ def parse_tf_plan(tf_plan_file: str, out_parsing_errors: Dict[str, str]) -> Tupl
     :type tf_plan_file: str - path to plan file
     :rtype: tf_definition dictionary and template_lines of the plan file
     """
-    tf_definition: Dict[str, Any] = {"resource": [], "data": []}
+    tf_definition: Dict[str, Any] = {"provider": [], "resource": [], "data": []}
```

Comment:
we are changing the tf_definition obj, can it broke something somewhere?

Reply:
I matched the tf_definition of the tf_plan to the regular terraform tf_definition, so I think it should not have any side effects.  
And of course all the existing tests are passing 🤞